network/network.go

```diff
@@ -276,6 +276,12 @@ func NewNetwork(
 		IPSigner:             peer.NewIPSigner(config.MyIPPort, config.TLSKey),
 	}
 
+	// Invariant: We delay the activation of durango during the TLS handshake to
+	// avoid gossiping any TLS certs that anyone else in the network may
+	// consider invalid. Recall that if a peer gossips an invalid cert, the
+	// connection is terminated.
+	durangoTime := version.GetDurangoTime(config.NetworkID)
+	durangoTimeWithClockSkew := durangoTime.Add(config.MaxClockDifference)
```

Comment:
q: this is another TODO: remove once Durango is activated right?

Reply:
added a comment